package.json

```diff
@@ -107,6 +121,5 @@
       "storybook": "start-storybook",
       "test": "tap 'src/**/*test.js' --no-coverage"
    },
-   "standard-version": {},
```

Comment:
That's actually intended. It's a placeholder for the config of `standard-version`, which takes care of writing my changelog with  the semantic commit messages.